rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/connectorwrapper/Worker.java

```diff
@@ -269,6 +269,7 @@ public synchronized void startTasks(Map<String, List<ConnectKeyValue>> taskConfi
                     }
                     producer.setMaxMessageSize(RuntimeConfigDefine.MAX_MESSAGE_SIZE);
                     producer.setLanguage(LanguageCode.JAVA);
+                    producer.setInstanceName(connectConfig.getRmqInstanceName());
```

Comment:
It seems that there's still only one instance will be created even though the instanceName config is set.

Reply:
I originally wanted to design like a replicator, but no utils class.  this pr #355

Reply:
Could we use one producer to send messages for all of the tasks? This can reduce the burden of brokers.

Reply:
yes，i will solve it on the #349